common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/databinding/ObservablePojoUtils.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.databinding;
+
+import java.beans.PropertyChangeListener;
+
+/**
+ * @author Andre Dietisheim
+ */
+public class ObservablePojoUtils {
```

Comment:
this looks like an anti pattern - why would one ever have a list of objects that one runs through where only some objects are observable ?

wouldn't that always be an error situation ?